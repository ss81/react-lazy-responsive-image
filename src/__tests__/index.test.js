import React from 'react';
import renderer from 'react-test-renderer';
import LazyResponsiveImage from '../index.js';

it('Rendes successfully the lazy image', () => {
    const tree = renderer
    .create(
        <LazyResponsiveImage
            src="foo@default.jpg"
            className="test-class"
            sources={{
                small: 'foo@small.jpg',
                medium: 'foo@medium.jpg',
                large: 'foo@large.jpg|foo@large-2x.jpg',
            }}
            breakpoints={{
                small: 600,
                medium: 900,
                large: 1200,
            }}
        />
    )
    .toJSON();
    expect(tree).toMatchSnapshot();
});