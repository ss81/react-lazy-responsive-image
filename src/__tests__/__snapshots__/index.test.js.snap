// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendes successfully the lazy image 1`] = `
<img
  className="b-lazy test-class "
  data-src="foo@default.jpg"
  data-src-large="foo@large.jpg|foo@large-2x.jpg"
  data-src-medium="foo@medium.jpg"
  data-src-small="foo@small.jpg"
  src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
/>
`;
